README.md

```diff
@@ -79,18 +79,18 @@ zk-zk-2                 1/1     Running            0          23s
 ```
 
 
-## Community, discussion, contribution, and support
+## Community, Discussion, Contribution, and Support
 
 Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
 
 You can reach the maintainers of this project at:
 
 - [Slack](http://slack.k8s.io/)
-- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-dev)
+- [Mailing List](https://groups.google.com/d/forum/kudobuilder)
```

Comment:
Whats the difference between?
`https://groups.google.com/d/forum/kudobuilder`
and
`https://groups.google.com/forum/#!forum/kudobuilder`